fmriprep/workflows/base.py

```diff
@@ -373,78 +381,116 @@ def init_single_subject_wf(subject_id, task_id, name,
+        if not func_only:  # Connect anatomical only if possible
+            func_preproc_wf = init_func_preproc_wf(
+                bold_file=bold_file,
+                layout=layout,
+                ignore=ignore,
+                freesurfer=freesurfer,
+                func_only=func_only,
+                use_bbr=use_bbr,
+                bold2t1w_dof=bold2t1w_dof,
+                reportlets_dir=reportlets_dir,
+                output_spaces=output_spaces,
+                template=template,
+                medial_surface_nan=medial_surface_nan,
+                output_dir=output_dir,
+                omp_nthreads=omp_nthreads,
+                low_mem=low_mem,
+                fmap_bspline=fmap_bspline,
+                fmap_demean=fmap_demean,
+                use_syn=use_syn,
+                force_syn=force_syn,
+                debug=debug,
+                output_grid_ref=output_grid_ref,
+                use_aroma=use_aroma,
+                ignore_aroma_err=ignore_aroma_err,
+            )
+
+            workflow.connect([
+                (anat_preproc_wf, func_preproc_wf,
+                 [('outputnode.t1_preproc', 'inputnode.t1_preproc'),
+                  ('outputnode.t1_brain', 'inputnode.t1_brain'),
+                  ('outputnode.t1_mask', 'inputnode.t1_mask'),
+                  ('outputnode.t1_seg', 'inputnode.t1_seg'),
+                  ('outputnode.t1_tpms', 'inputnode.t1_tpms'),
+                  ('outputnode.t1_2_mni_forward_transform',
+                   'inputnode.t1_2_mni_forward_transform'),
+                  ('outputnode.t1_2_mni_reverse_transform',
+                   'inputnode.t1_2_mni_reverse_transform'),
+                  # Undefined if --no-freesurfer, but this is safe
+                  ('outputnode.subjects_dir', 'inputnode.subjects_dir'),
+                  ('outputnode.subject_id', 'inputnode.subject_id'),
+                  ('outputnode.t1_2_fsnative_forward_transform',
+                   'inputnode.t1_2_fsnative_forward_transform'),
+                  ('outputnode.t1_2_fsnative_reverse_transform',
+                   'inputnode.t1_2_fsnative_reverse_transform')]),
+            ])
+        else:
+            func_preproc_wf = init_onlyfunc_preproc_wf(
+                bold_file=bold_file,
+                layout=layout,
+                ignore=ignore,
+                reportlets_dir=reportlets_dir,
+                output_dir=output_dir,
+                omp_nthreads=omp_nthreads,
+                low_mem=low_mem,
+                fmap_bspline=fmap_bspline,
+                fmap_demean=fmap_demean,
+                debug=debug,
+                use_aroma=use_aroma,
+                ignore_aroma_err=ignore_aroma_err,
+            )
+
+            workflow.connect([
+                (inputnode, func_preproc_wf, [('subjects_dir', 'inputnode.subjects_dir')]),
+                (bidssrc, bids_info, [
+                    (('bold', fix_multi_bold_source_name), 'in_file')]),
```

Comment:
`fix_multi_bold_source_name` is undefined

Reply:
I assumed there was an analog to fic_multi_t1w_source_name, which pops up in the anat_only workflow.